_posts/2023-09-23-dd_00006_full_adder.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: single
+title: Combinational Circuits - The Full-Adder
+category: Digital Design
+permalink: /categories/digital_design/full_adder
+excerpt: "The heart of digital adders."
+description:
+  "This post introduces the Full-Adder, an extension of the Half-Adder."
+author: Kirthihan Yasotharan
+toc: false
+toc_label: "Content"
+toc_sticky: true
+toc_icon: "align-justify"
+header:
+  overlay_color: "#000"
+  overlay_filter: "0.5"
+  overlay_image: /assets/images/digital_design.jpeg
+---
+
+<div markdown='1' class='note-normal'>
+## TLDR
```

Comment:
Idea was to try out having the summary already at the beginning, so that readers can already see what to expect from the post. I have changed the title to "key takeaways" now.

What do you think?

Reply:
better!